WordPress/WordPressUITests/Screens/MeTabScreen.swift

```diff
@@ -20,7 +23,7 @@ class MeTabScreen: BaseScreen {
         notificationSettingsButton = app.cells["notificationSettings"]
         doneButton = app.navigationBars.buttons["doneBarButton"]
 
-        super.init(element: appSettingsButton)
+        try super.init(element: appSettingsButton)
```

Comment:
This is it, the changes between `BaseScreen` and `ScreenObject` ([see implementation](https://github.com/Automattic/ScreenObject/blob/trunk/Sources/ScreenObject/ScreenObject.swift)) are minimal at this point.

The main thing I did was removing the forced `try` from it, as it didn't seem appropriate for a standalone framework to brutally crash its consumers 😅